src/Facebook/Object/User.hs

```diff
@@ -33,6 +33,14 @@ import Facebook.Object.Checkin
 -- /NOTE:/ We still don't support all fields supported by
 -- Facebook. Please fill an issue if you need access to any other
 -- fields.
+data Picture =
```

Comment:
The `Picture` data type is declared between the comment for `User` and the `User` datatype itself.

The `Picture` data type should either be declared above the comment for `User`, or below the `User` datatype.  It would be best if it had a comment as well.